<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFlowGraphTokensDlg</class>
 <widget class="QDialog" name="CFlowGraphTokensDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Tokens</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="GRAPH_TOKEN_LIST"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="ADD_GRAPHTOKEN">
       <property name="text">
        <string>New Token</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DELETE_GRAPHTOKEN">
       <property name="text">
        <string>Delete Token</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EDIT_GRAPHTOKEN">
       <property name="text">
        <string>Edit Token</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
